### DISCUSSION TITLE
Compiling only current file

### DISCUSSION THREAD

**Discussion by paubanon** [texstudio-org/texstudio#3610] [Q&A]
I've noticed that when I press F5 to Build&Compile, all files that I have opened in tabs get compiled. 

That makes the compilation slower and consumes more battery than expected.
Is there a way of setting TexStudio to compile only the current active tab?

**Comment by sunderme** [comment]
Only the root file of the current document gets compiled.
What do you mean exactly by "all" ?

**Reply by paubanon** [reply]
If I have 5 .tex files opened in tabs, all 5 files get compiled

**Comment by octaeder** [comment]
we need more clarification about dependencies between the files. Do these files build one final pdf or are there 5 different pdfs?

**Comment by sunderme** [comment]
I would rather like to see screen cast/screen shots.
txs does not compile multiple documents.
